### DISCUSSION TITLE
MailMate

### DISCUSSION THREAD

**Discussion by imjma** [elrumo/macOS_Big_Sur_icons_replacements#1171] [Icon request] [answered]
Anyone can make Mailmate one? https://blog.freron.com

Thanks

**Comment by blaichch** [comment, marked as answer]
How about this one?

<img width="512" alt="MailMate" src="https://user-images.githubusercontent.com/12624897/118522598-860f4300-b73c-11eb-8464-6c3fa1abdb29.png">
